spec/replication/ar_pglogical_spec.rb

```diff
@@ -71,6 +71,23 @@
         end
       end
 
+      describe "#node_dsn_update" do
+        let(:new_dsn) { "host='newhost.example.com' dbname='vmdb_test' user='root'" }
+
+        it "sets the dsn" do
+          connection.pglogical.node_dsn_update(node_name, new_dsn)
+          dsn = connection.exec_query(<<-SQL).first["if_dsn"]
+          SELECT if_dsn
```

Comment:
Can you indent this one more so it's clearer that it's part of the dsn variable...leave the closing `SQL` where it is...then it looks like a block.